### ISSUE TITLE
Create season stats table and populate this onto the Players Page

### ISSUE THREAD

**Issue by jcgoodru** [RyanDanielOMara/nbai#116]
(1) Create a ```player_season_stats``` table.  The object will preferably be named PlayerSeasonStatsRecord (or something like that).  This will essentially be a clone of ```player_game_logs```, but will aggregate stats across a season instead of per-game.  Some of the fields will need to be removed (like games_dict and team_id and stuff like that).  The index of this table should be player_id, and should be non-unique.  This table keeps track of the total number of (pts reb ast blk stl etc) for a given player in a given season, as well as the number of games they played in.

(2) Create a function in ```database_util.py``` that populates this info into the table.  You can pass in the player_game_logs list and populate it via those nodes.

First, get all the player_ids from player_game_nodes
```
pids = list(set(p.player_id for p in player_game_nodes))
```

Next, get all the season stat records for each of these players.  If there isn't one, create a new one for this player.
```
rec = connection.PlayerSeasonStatsRecord.one({ f.player_id : pid, f.year : year })
if rec is None:
    rec = connection.PlayerSeasonStatsRecord()
    rec.player_id = player_id
```

Next, zero out everyone's stats.
```
   rec.pts = 0
   rec.etcetera = 0
```

After doing this, iterate through the game nodes and add that nodes' stats to that players' record.

Save all the records after doing this (after iterating through the entire list of game_logs).
```
rec.save()
```

**Comment by jcgoodru** [comment]
Partially resolved with #141 .  That commit built the table that houses the necessary data.

Only thing that needs to be done is grabbing the player from the database, iterating across the records ,and building the table rows:
```
def get_player_page_table(player_id):
    data = []
    for rec in connection.PlayerSeasonStatsRecord.find({ f.player_id : player_id }):
        build_table_rows
    return data
```
This should take very little critical thinking.

**Comment by jcgoodru** [comment]
This is done right?  Reopen if not
